packages/metals-vscode/package.json

```diff
@@ -1085,7 +1105,8 @@
     "metals-languageclient": "file:../metals-languageclient",
     "promisify-child-process": "4.1.1",
     "semver": "^7.5.2",
-    "vscode-languageclient": "8.1.0"
+    "vscode-languageclient": "8.1.0",
+    "vscode-extension-telemetry-wrapper": "^0.13.3"
```

Comment:
Is this needed?

Reply:
Good point, `registerCommand` would do the same